modules/gateway.go

```diff
@@ -13,6 +13,7 @@ var (
 	BootstrapPeers = []NetAddress{
 		"23.239.14.98:9981",
 		"87.98.216.46:9981",
+		"45.79.132.35:9981",
```

Comment:
ping these and check that they are port forwarded + online before you add them:

 68.55.10.144 
109.206.33.225
79.51.183.132
83.211.176.23

they are all volunteers and are aware that boostrap nodes see more traffic than other nodes